src/pages/Dishes/DishList.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { useState, useEffect } from "react";
+import {   RouteComponentProps } from "react-router-dom";
+
+import Card from "../../components/Cards/Card";
+import styled from "styled-components";
+import firebase from "firebase";
+
+
+function DishList({ match }: RouteComponentProps<any>) {
+    const [dishes, setDishes] = useState<any[]>([])
+
+    useEffect(() => {
+      const fetchData = async () => {
+        const db = firebase.firestore();
+        let restaurantsRef = db.collection('restaurants').doc(`${match.params.id}`).collection('dishes');;
+        let allRestaurants = await restaurantsRef.get()
+        let dataArray: any = []
+        for (const doc of allRestaurants.docs) {
+          dataArray.push({ ...doc.data(), id: doc.id })
+        }
+        setDishes(dataArray)
+
+      };
+      fetchData();
+    }, []);
+
+    console.log(dishes)
```

Comment:
Before you push your commits, try cleaning up the code and removing unnecessary variables and console logs - that way your code can be more production ready :)